control-plane/subcommand/sync-catalog/command.go

```diff
@@ -156,8 +156,8 @@ func (c *Command) init() {
 			"\"debug\", \"info\", \"warn\", and \"error\".")
 	c.flags.BoolVar(&c.flagLogJSON, "log-json", false,
 		"Enable or disable JSON output format for logging.")
-	c.flags.StringVar(&c.flagPurgeK8SServicesFromNode, "purge-k8s-services-from-node", "",
-		"Specifies the name of the Consul node for which to deregister synced Kubernetes services.")
+	// c.flags.StringVar(&c.flagPurgeK8SServicesFromNode, "purge-k8s-services-from-node", "",
+	// "Specifies the name of the Consul node for which to deregister synced Kubernetes services.")
```

Comment:
We should remove instead of just comment out, right?

Reply:
yeah, we're going to be putting it back with some different behavior for  the next patch but it's probably better to just remove it now